### PR TITLE
Add r-rcppnumerical

### PR DESCRIPTION
**Pull request by dpryan79**

### CODE REVIEW

File: recipes/r-rcppnumerical/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  commands:
+    - $R -e "library('RcppNumerical')"           # [not win]
+    - "\"%R%\" -e \"library('RcppNumerical')\""  # [win]
+
+about:
+  home: https://github.com/yixuan/RcppNumerical
+  license: GPL (>= 2)
+  summary: A collection of open source libraries for numerical computing (numerical integration,
+    optimization, etc.) and their integration with 'Rcpp'.
+  license_family: GPL3
```
Comment (bgruening) [inline recipes/r-rcppnumerical/meta.yaml]: xref: https://github.com/yixuan/RcppNumerical/issues/9
If it is GPL we need to ship a license file ... the recipe looks ok

File: recipes/r-rcppnumerical/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+extra:
+  recipe-maintainers:
+    - dpryan79
```
Comment (cbrueffer) [inline recipes/r-rcppnumerical/meta.yaml]: Could you add more maintainers?
```
    - johanneskoester
    - bgruening
    - daler
    - jdblischak
    - cbrueffer
    - dbast
```

Comment (dpryan79) [reply recipes/r-rcppnumerical/meta.yaml]: Certainly!

File: recipes/r-rcppnumerical/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/r-rcppnumerical/meta.yaml]: Actually, I expect that this is why the CRAN skeleton no longer adds a license file line in.

**Pull request merged**